packages/types/actions.ts

```diff
@@ -13,7 +13,8 @@ export type TAction = z.infer<typeof ZAction>;
 
 export const ZActionInput = z.object({
   environmentId: z.string().cuid2(),
-  sessionId: z.string().cuid2(),
+  personId: z.string(),
```

Comment:
personId is new and now required for actions but previously was not needed; meaning validation of input in actions endpoint would throw an error?